### ISSUE TITLE
All commands return 'not found' when running extension in WSL mode ( 'Open Remote - WSL')

### ISSUE THREAD

**Issue by hemulya** [dbcodeio/public#127]
Hello devs,

I have an issue of working dbcode extension in my setup
- VSCodium - 1.95.3 
- Open Remote - WSL - 0.0.4

In WSL mode extension returns errors like this
```
Error running command dbcode.connections.add: command 'dbcode.connections.add' not found. This is likely caused by the extension that contributes dbcode.connections.add.
```
All dbcode extension commands return the same message. Only commands names are different.

All working good in non WSL mode.

I know that WSL extension looks very raw but this is only one option to run WSL in Open Source VSCodium. I will be appreciated, if you find time to look to my issue.

Thanks

**Comment by mikeburgh** [comment]
I can look into it, guessing it's going to be something todo with that open remote - wsl extension and how it's mapping the extension and host. 

Just to confirm, this is the extension you are refering to ? https://github.com/jeanp413/open-remote-wsl

Also, to confirm your setup, you are in Windows, running VSCodium, and trying to use the WSL remote (linux on windows)

**Comment by hemulya** [comment]
> Just to confirm, this is the extension you are refering to ? https://github.com/jeanp413/open-remote-wsl

yes, that's one.

> Also, to confirm your setup, you are in Windows, running VSCodium, and trying to use the WSL remote (linux on windows)

Exactly!

I'm not sure what extension cause the issue. I have researched issue before and looks like fixes mostly were in extension which commands belongs to. I hope this is an easy fix if it is there!

Thank you for your time!

**Comment by mikeburgh** [comment]
Just got some time to dig into this.. I was able to get it to work without any issues.

Here is what I did.

1. Fresh windows install
2. Fresh Ubuntu 24.04.1 install
3. Enabled SSH in WSL based on: https://jmmv.dev/2022/02/wsl-ssh-access.html (first 5 steps)
4. Installed VsCodium
5. Installed open-remote-wsl connection
6. Configured ssh config for new host: 
Host wsl
    HostName localhost
    Port 2022


My first attempt to connect to WSL it failed to install codium in WSL, second attempt it was fine, and then after that I could install the extension, and connect to the sample database an open a table, see (window vscode is the background, wsl based one in the front):

<img width="1950" alt="Image" src="https://github.com/user-attachments/assets/5917a063-1aa8-4e63-936d-21a0e2d7b953" />

Is there anything different in your config ? Or I wonder if you have tried the latest VSCodium/DBCode extensions to see if it's fixed.

**Comment by ya-sharaf** [comment]
Yeah! It works now! Looks like an issue were in IDE and fixed in latest updates. I hope it wasn't take too much time to investigate. Thank you!

**Comment by mikeburgh** [comment]
Good news, and nope, not much time at all.. so a good outcome!
